addons/mail/static/src/messaging/component/ellipsis/ellipsis.scss

```diff
@@ -0,0 +1,84 @@
+
+// ------------------------------------------------------------------
+// Layout
+// ------------------------------------------------------------------
+
+.o_Ellipsis {
+    display: flex;
+    align-items: center;
+}
+
+.o_Ellipsis_dot {
+    display: flex;
+    flex: 0 0 auto;
+}
+
+.o_Ellipsis_separator {
+    min-width: 1px;
+    flex: 1 0 auto;
+}
+
+// ------------------------------------------------------------------
+// Style
+// ------------------------------------------------------------------
+
+.o_Ellipsis_dot {
+    border-radius: 50%;
+    background: gray('500');
+
+    &.o-sizeMedium {
+        width: 5px;
+        height: 5px;
+    }
+
+    &.o-sizeSmall {
+        width: 3px;
+        height: 3px;
+    }
```

Comment:
I feel these would be better if they were a factor of `$font-size-base` somehow?

Reply:
Doesn't really work, because shapes dimensions become decimals and are auto-rounded so we have inconsistencies like this:
![inconsistent-typing-ellipsis](https://user-images.githubusercontent.com/6569390/82827881-3347d600-9eb0-11ea-956d-0b6a49fc6741.png)

Reply:
Even with `floor()`?

Reply:
`floor(0.99999rem)` = `0rem`

Reply:
Oh yeah it's not px... Ok I give up on this one for now.